paper-badge.html

```diff
@@ -79,7 +87,14 @@
       }
     </style>
 
-    <div id="badge">{{label}}</div>
+    <div id="badge">
```

Comment:
Do you need this wrapping `badge` div? the style can just go in the `:host`, no? (if you really do need it, then it can be a class and not an `id` since it's not being used anywhere other than styling)